com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/MaxRectsLayoutStrategy.java

```diff
@@ -119,14 +120,14 @@ private Page packPage(ArrayList<RectNode> inputRects) {
             BinarySearch sizeSearch = new BinarySearch(minSize, maxSize);
             int size = sizeSearch.reset();
             while (size != -1) {
-                Page result = packAtSize(true, size, size, inputRects);
+                Page result = packAtSize(true, size - settings.paddingX, size - settings.paddingY, inputRects);
```

Comment:
Reduce all the places where we use size for `margin` to be able to move all the rects from top left corner for `margin` count of pixels